src/hope_dedup_engine/apps/api/admin/hdetoken.py

```diff
@@ -1,8 +1,20 @@
 from django.contrib.admin import ModelAdmin, register
+from django.contrib import messages
 
 from hope_dedup_engine.apps.api.models import HDEToken
 
 
 @register(HDEToken)
 class HDETokenAdmin(ModelAdmin):
-    list_display = ("key", "user")
+    list_display = ("user",)
+    readonly_fields = ("key",)
```

Comment:
it should have editable=false in the model definition and not listed in readonly_fields